Polishchuk_Dariia_HW2(3 reverse).cpp

```diff
@@ -0,0 +1,15 @@
+#include <iostream>
+
+int main() {
+  int number{};
+  int reverse_number = 0;
+  std::cout << "Insert the number: ";
+  std::cin >> number;
+  while (number) {
```

Comment:
```suggestion
  while (number > 0) {
```
просто как предложение для читабельности